### ISSUE TITLE
autoCenterAndZoom doesn't work well with world-wide points

### ISSUE THREAD

**Issue by CIPowell** [mapstraction/mxn#84]
When there are markers above and below the equator and left and right of the meridian, auto Center and zoom seems to ignore some of them and zoom in on one point;

In this case, with markers in Europe only it works fine, but with markers in Europe, Africa and Oceana (Australia, New Zealand and Singapore) the map zooms in on Southern Africa.

Using googlemapsv3.

set up code

```
            map = new mxn.Mapstraction("map", "googlev3");
    map.setCenterAndZoom(new mxn.LatLonPoint(0,0), 1);
    map.addLargeControls();
    map.addMapTypeControls();
```

markers are added using addMarkerWithData.

**Comment by brunob** [comment]
Hi, tested here and it works without any problem :

http://labo.eliaz.fr/spip.php?article85

Could you give us the url of your problematic page ?

**Comment by CIPowell** [comment]
http://www.spatialepidemiology.net/snp/mxnDebug.html

Here's an example page with some test data.

**Comment by brunob** [comment]
Ok i got it, there is a bug in autoCenterAndZoom(), i'll make a pull request for the patch.

++

**Comment by brunob** [comment]
So, i've pushed a branch for that in my repo. I've started bb15c44961aa3827a57a586bd3edca9968a4dc26 refactoring autocenterandzoom code by using bounds and bounds.extends to avoid redundancy code in mxn.core.

The bug is still there but now we know that we have to fix bounds.extend()...

**Comment by brunob** [comment]
So, i've tested many thing during all day and i can't get it working well for now. I've looked at opelayers code and they do the same thing that mapstration, so your data should not work with a native openlayers map. I've also tried your data on a native google maps v3 and it works if i use map.fitBounds(bounds); to autocenterandzoom. The problem is that i can't find (and not sure that it's possible) the source code of google maps api...

Stay tuned ++
